packages/inventory/src/components/table/helpers.js

```diff
@@ -18,11 +18,11 @@ export const buildCells = (item, columns, extra) => {
     });
 };
 
-export const createRows = (rows = [], columns = [], { actions, expandable, ...extra } = {}) => {
+export const createRows = (rows = [], columns = [], { actions, expandable, customNoSystemsTable, ...extra } = {}) => {
     if (rows.length === 0) {
         return [{
             cells: [{
-                title: <NoSystemsTable />,
+                title: customNoSystemsTable ? customNoSystemsTable : <NoSystemsTable />,
```

Comment:
The same as in the previous PR, we can skip `custom` from props and make the default component as the default prop.